### PR TITLE
Aiat 185 end to end scenarios

### PR DESCRIPTION
**Pull request by CallumRobb-ScottLogic**
Additional unique link questions being created, answered, the results page checked, and then deleted.

### CODE REVIEW

File: playwright/tests/treeTestResultsTests.spec.js [hunk 1]
```diff
@@ -61,4 +61,231 @@ test('Unique link results page test', async ({ studiesPage, resultsPage, editTre
     //Delete Studuies
     await studiesPage.goto();
     await studiesPage.deleteStudy(testTitle);
+});
+
+test('Unique link give up results page test', async ({ studiesPage, resultsPage, editTreeTestPage, treeTestStudy}) => {
```
Comment (kclark-scottlogic) [inline playwright/tests/treeTestResultsTests.spec.js:66]: Would these not be best in the uniqueLink.spec file? or if these are the big e2e scenarios have a e2e.spec file of their own?

File: playwright/tests/treeTestResultsTests.spec.js [hunk 2]
```diff
@@ -61,4 +61,231 @@ test('Unique link results page test', async ({ studiesPage, resultsPage, editTre
...
+    await resultsPage.responsesByTask('Where is the Apples?');
+    await expect(resultsPage.taskCell).toContainText(answerOne);
+    await expect(resultsPage.taskCell).toContainText('1 of 1');
+
+    await resultsPage.responsesByTask('Where is the Bacon?');
+    await expect(resultsPage.taskCell).toContainText(answerTwo);
+    await expect(resultsPage.taskCell).toContainText('1 of 1');
+
+        //Delete Studuies
+        await studiesPage.goto();
```
Comment (kclark-scottlogic) [inline playwright/tests/treeTestResultsTests.spec.js]: Should this be indented like this?

Comment (CallumRobb-ScottLogic) [reply playwright/tests/treeTestResultsTests.spec.js]: No it should not, fixed it

File: playwright/tests/treeTestResultsTests.spec.js (hunk 1 above)
Comment (CallumRobb-ScottLogic) [reply playwright/tests/treeTestResultsTests.spec.js:66]: I think they should remain in the results page, as it is ultimately the final check that this has worked.

**Review by kclark-scottlogic — APPROVED** [review]
Other than code clean up which we discussed for a later date. This looks good to me

**Pull request merged**